truckersmp_cli/configfile.py

```diff
@@ -0,0 +1,44 @@
+"""
+Configuration file handler for truckersmp-cli main script.
+
+Licensed under MIT.
+"""
+
+import configparser
+import os
+
+
+class ConfigFile:
+    """Configuration file."""
+
+    def __init__(self, configfile):
+        """
+        Initialize ConfigFile object.
+
+        configfile: Path to configuration file
+        """
+        self._thirdparty_total_wait = 0
+        self._thirdparty_executables = []
+        parser = configparser.ConfigParser()
+        parser.read(configfile)
+        for sect in parser.sections():
+            # get data from valid thirdparty.* sections
+            if not sect.startswith("thirdparty.") or "executable" not in parser[sect]:
+                continue
+            try:
+                wait = int(parser[sect]["wait"])
+                self._thirdparty_total_wait += wait
```

Comment:
Doesn't using the max value here makes more sense?

If progA needs 20s wait and progB 30s wait and we're starting all at once we've finished the waiting time for progA already in waiting for progB.